[Finish] Improving German localization

enterih opened the pull request:
I found some stuff to improve the translation. I'm not done yet but i think it's ready to get an eye on it. 

Hope it will be used :)

Greets Sprungente/enterih

enterih committed:
Improving 

I found some stuff to improve the translation. I'm not done yet but i think it's ready to get an eye on it. 

Hope it will be used :)

Greets Sprungente/enterih

enterih committed:
mistake^^

^

enterih committed:
Part 2

Adding more Improvements
Hope it helps other serverowners^^

enterih committed:
Part 2 fixes

mdcfe commented on Essentials/src/messages_de.properties:
```diff
@@ -127,18 +127,18 @@ extinguishOthers=\u00a76Du hast {0}\u00a76 gel\u00f6scht.
 failedToCreateConfig=Fehler beim Erstellen der Konfiguration {0}.
 failedToWriteConfig=Fehler beim Schreiben der Konfiguration {0}.
-false=\u00a74falsch\u00a7r
+false=\u00a74Nein\u00a7r
```
Is this actually more appropriate than `falsch`?

enterih replied:
in situation like /whois 
yes a lot :)

enterih commented:
Yes i think
True is translated with „ja“. I recommend also change true and false into yes and no

In my opinion it sounds in the most situation like /whois better.

Please don‘t merge it. It isn‘t finished yet

Am 08.10.2018 um 10:46 schrieb md678685 <notifications@github.com<mailto:notifications@github.com>>:


@md678685 commented on this pull request.

________________________________

In Essentials/src/messages_de.properties<https://github.com/EssentialsX/Essentials/pull/2198#discussion_r223288208>:

> @@ -127,18 +127,18 @@ extinguishOthers=\u00a76Du hast {0}\u00a76 gel\u00f6scht.
 failedToCloseConfig=Fehler beim Schlie\u00dfen der Konfiguration {0}.
 failedToCreateConfig=Fehler beim Erstellen der Konfiguration {0}.
 failedToWriteConfig=Fehler beim Schreiben der Konfiguration {0}.
-false=\u00a74falsch\u00a7r
+false=\u00a74Nein\u00a7r


Is this actually more appropriate than falsch?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/EssentialsX/Essentials/pull/2198#pullrequestreview-162375549>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AOpna2NUU-A4hZeV4LdZeWEtjr-WGZ97ks5uixEBgaJpZM4XHT3c>.

enterih committed:
Part 3 



Notice for me 
a-d finished
\u00f6 ö    \u00e4 ä  \u00fc ü

enterih committed:
Part 4

Notice for me 
a-i finished
\u00f6 ö    \u00e4 ä  \u00fc ü

mdcfe reviewed (dismissed):
I'm happy to merge this when you're ready.

enterih committed:
Last part 

is line 456
requiredBukkit=\u00a76* \! * Du brauchst mindestens CraftBukkit Build \#{0}, lade es von http\://dl.bukkit.org/downloads/craftbukkit/
still exist? If yes, could a reference to spigot be added?

It's ready to merge. Can i take care on it in the future?

enterih committed:
small polishing

enterih committed:
Update messages_de.properties

enterih committed:
Update messages_de.properties

enterih commented:
@md678685 it's ready to merge
I wanna maintanance in the future if it's ok for you.

enterih committed:
Adding new strings 

added strings from #2278

mdcfe committed:
Merge branch '1.13' into patch-2
